src/app/core/record-emails/record-emails.service.ts

```diff
@@ -49,8 +49,8 @@ export class RecordEmailsService {
     } else if (!options.forceReload) {
       return this.$emailsSubject
     }
-    if (options.cleanCacheIfExist && this.$emailsSubject) {
-      this.$emailsSubject.next(<EmailsEndpoint>undefined)
+    if (options.cleanCacheIfExist) {
```

Comment:
@DanielPalafox please check this, to call `next` on the `this.$emailsSubject` it must not be null. So this should not make a big difference although checking if the `this.$emailsSubject` is not really require since is created on line 48